web/locales/de.json

```diff
@@ -6,6 +6,29 @@
         "datasources": "Datenquellen",
         "contribute": "Wirke mit, indem du <a href=\"%s\" target=\"_blank\">dein Land hinzufügst</a>"
     },
+    "mobile-info-tab": {
+      "map-info-text": "Diese Karte zeigt in Echtzeit wie viel CO2 ausgestoßen wird. Wir färben alle Gegenden der Welt mit ihrer Kohlenstoffintensität (einschließlich aller Treibhausgase) der dort verbrauchten Elektrizität. Grünere Farben bedeuten Klima freundlichere Elektrizität.",
+      "electricity-origins-text": "Die Pfeile auf der Karte zeigen die Ströme der Elektrizität zwischen verschiedenen Gegenden. Klicke auf eine Gegend um mehr Informationen über die Herkunft der Elektrizität zusehen.",
+      "wind-solar-text": "Wind- und Solarenergie sind abhängig vom Wetter. Klicke auf den Sonnen- und Windknopf um die Echtzeit Stärke des Windes und des Sonnenscheins überall in der Welt anzuzeigen."
+    },
+    "onboarding-modal": {
+      "view1": {
+        "header": "Die Elektrizitätskarte",
+        "subtitle": "Ausweisung der Folgen der Elektrizität am Klima"
+      },
+      "view2": {
+        "header": "Sieh in Echtzeit wieviel CO2 ausgestoßen wird um deine Elektrizität zuproduzieren",
+        "text": "Wir färben alle Gegenden der Welt mit ihrer Kohlenstoffintensität (einschließlich aller Treibhausgase) der dort verbrauchten Elektrizität. Grünere Farben bedeuten Klima freundlichere Elektrizität."
```

Comment:
The current German is:

            "header": "Betrachte in Echtzeit, wie viel CO2-Äquivalente bei der Stromerzeugung freigesetzt werden.",
            "text": "Die Färbung der Zonen erfolgt anhand der spezifischen Emissionen klimawirksamer Gase (CO2-Intensität), die zur Deckung des Strombedarfs in einer Region freigesetzt werden. Eine grüne Färbung steht dabei für eine klimafreundlichere Erzeugung."